tools/ml-dataprep/bigquery.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module containing a helper class to interact with BigQuery."""
+
+
+from google.cloud import bigquery
+import queries
+
+
+class BqClient:
+
+    """Helper class to run BigQuery queries and export BigQuery tables."""
+
+    def __init__(self):
+        """Default constructor initializing the BigQuery client."""
+        self._bq_client = bigquery.Client()
```

Comment:
It would be useful to leave the option of using explicit alternative credential instead of environment default.

Reply:
Done in 41b50ecb64c32a73982fa74faa6248331a47e789